ROADMAP.md

```diff
@@ -0,0 +1,54 @@
+# Metal3 Roadmap
+
+The Metal3 Roadmap is maintained as a Github project and can be found
+[here](https://github.com/orgs/metal3-io/projects/2).
+
+## Description
+
+Each column in the project represents the work items for a specific release of
+either Baremetal Operator or Cluster API Provider Metal3. In addition there is
```

Comment:
Weirdly the Metal3 roadmap is CAPM3 or BMO... Either we need to re-work the projects per the recently discussed "release layers", or maybe even more preferably the project as a whole, and then per repo/layer separately.

Reply:
Yes those are outdated